draft-ietf-gnap-core-protocol.md

```diff
@@ -204,29 +194,37 @@ role of the AS as defined by the protocol.
 In addition to the roles above, the protocol also involves several 
 elements that are acted upon by the roles throughout the process.
 
+Attribute
+: characteristics related to an end-user.
+
 Access Token
-: A credential representing a set of access rights
-delegated to the client instance. The access token is created by the AS, consumed
-and verified by the RS, and issued to and carried by the client instance. The contents
-and format of the access token are opaque to the client.
+: digitally signed data that contains specific rights and/or attributes.
+
+    Note: an access token can be first issued to an end-user (usually requiring his authentication) and subsequently refreshed.
+
+Claim
+: statement about a subject.
 
 Grant
-: The process by which the client instance requests and is given delegated
-access to the RS by the AS through the authority of the RO.
+: (verb): to permit an end-user to exercise some rights and/or to assert some attributes at a specific time and during a specific duration. (noun): the act of granting.
 
-Cryptographic Key
-: A cryptographic element binding a request to a
-holder of the key. Access tokens and client instances can be associated with
-specific keys.
+Privilege
+: right or attribute associated with an end-user.
 
 Resource
-: A protected API served by the RS and accessed by the client instance. Access to this
-resource is delegated by the RO as part of the grant process.
+: public or protected resource from a RS.
+
+Protected Resource
+: protected API (Application Programming Interface) served by a RS and that can be accessed by a client, if and only if a valid access token is provided.
```

Comment:
What's the value of separating "resource" and "protected resource"? Do we have an "unprotected resource" in scope?

Reply:
No, but then we probably should always refer to protected resource everywhere?

Reply:
Yes, it probably should either be protected resource or, most likely, RS everywhere as that's what the protocol deals with.

Reply:
ok, proposed change: 

Resource Server (RS)
: server that provides operations on protected resources where operations require a valid access token issued by an AS trusted by the RS for this purpose.

+ keep protected resource definition
- remove resource definition

Will have to check in the main text that we refer to the RS or protected resources

Reply:
When doing it,this creates long/convoluted sentences -> I just add a note "Note: to avoid complex sentences, the specification document may simply refer to resource instead of protected resource."